app/policies/assignment_policy.rb

```diff
@@ -1,18 +1,21 @@
 class AssignmentPolicy < ApplicationPolicy
   class Scope < ApplicationScope
     def resolve
-      all if superadmin?
+      return all if superadmin?
+      scope.where(creator_id: user.id) if department_manager?
```

Comment:
I think (based on reviews of other PRs) that there should be a last case where it returns `none`. That concerns also the other policy files touched in this PR.

Reply:
Yeah I think that would make sense :)